plugins/telemetry-otel/src/internalClusterTest/java/org/opensearch/telemetry/tracing/RTCTracingDisabledSanityIT.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.tracing;
+
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import org.opensearch.client.Client;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.telemetry.OTelTelemetryPlugin;
+import org.opensearch.telemetry.TelemetrySettings;
+import org.opensearch.test.OpenSearchIntegTestCase;
+
+import static org.opensearch.index.query.QueryBuilders.queryStringQuery;
+
+@OpenSearchIntegTestCase.ClusterScope(scope = OpenSearchIntegTestCase.Scope.TEST, supportsDedicatedMasters = false, minNumDataNodes = 2)
+public class RTCTracingDisabledSanityIT extends OpenSearchIntegTestCase {
```

Comment:
Apologies, what `RTC` stands for?

Reply:
Thanks for pointing this out. Renamed it to TelemetryTracerEnabledSanityIT.java